src/scgpt/pad_tokenize/test.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+import sys
+import mudata as mu
+import numpy as np
+from scgpt.tokenizer.gene_tokenizer import GeneVocab
+from scgpt.preprocess import Preprocessor
+
+## VIASH START
+meta = {
+    "resources_dir": "resources_test/scgpt",
+    "executable": "./target/docker/scgpt/integration_pad_tokenize/integration_pad_tokenize",
+    "temp_dir": "tmp",
+    "config": "./target/docker/scgpt/integration_pad_tokenize/.config.vsh.yaml"
+}
+## VIASH END
+
+input = f"{meta['resources_dir']}/scgpt/test_resources/Kim2020_Lung_subset.h5mu"
+vocab_file = f"{meta['resources_dir']}/scgpt/source/vocab.json"
+input_file = mu.read(input)
+
+## START TEMPORARY WORKAROUND DATA PREPROCESSING
```

Comment:
I think you copied this `START TEMPORARY...` from other scripts. The reason why it was used there because it was not possible to specify shared code (in the config using `test_resources:`) to be used across multiple components by using`sys.path.append(meta['resources_dir'])` on run environments that use nextflow fusion. 

I would not use this tag here because we are probably going to use a regex to remove the lines of code automatically when we fix this in viash.

I think here you want to either:
(1) Save the intermediary result of the preproc to resources test (and s3) and use it directly.
(2) Add some dummy output to be used as a test.

Reply:
Had to implement this because padding and tokenizing can only be done once certain other scGPT-specific pre-processing steps are performed, but those components are not merged into the scGPT branch yet. So my idea would be to have all these components merged into scGPT and then update the scGPT_test_resources.sh script, to save intermediary temporary data and avoid pushing too many files to s3 (https://github.com/openpipelines-bio/openpipeline/issues/780)... WDYT?

Reply:
Sounds good!